tests/cases/py_client/lib_test.py

```diff
@@ -92,6 +92,19 @@ def tearDown(self):
 
         base.TestCase.tearDown(self)
 
+    def testSession(self):
+        @httmock.urlmatch(path=r'.*/describe$')
```

Comment:
It looks like the path it's matching against is `/api/v1/describe`. Do we want to hardcode this, or maybe use the version from `girder.constants`? (I copied this usage of `httmock` from elsewhere in the same file).